src/store/audio.ts

```diff
@@ -21,6 +21,25 @@ async function generateUniqueId(audioItem: AudioItem) {
     .join("");
 }
 
+function parseTextFile(body: string, charactorInfos?: CharactorInfo[]): AudioItem[] {
+  const charactors = new Map(charactorInfos?.map((info, index) => [info.metas.name, index]));
+  if(!charactors.size) return [];
+
+  const audioItems: AudioItem[] = [];
+  const seps = [',', '\n'];
+  let lastCharactorIndex: number | undefined = 0;
```

Comment:
ここundefined入ることなさそうなので、消しても良いかも？

```suggestion
  let lastCharactorIndex: number = 0;
```